src/utils/time.ts

```diff
@@ -1,17 +1,50 @@
-import { format, formatDuration, formatRelative } from "date-fns";
-
-import { VenueEvent } from "types/venues";
+import {
+  differenceInSeconds,
+  format,
+  formatDuration,
+  formatRelative,
+  fromUnixTime,
+  getUnixTime,
+  intervalToDuration,
+  isBefore,
+  startOfDay,
+} from "date-fns";
 
+/**
+ * @deprecated in favor of using date-fns functions
+ */
 export const ONE_SECOND_IN_MILLISECONDS = 1000;
+/**
+ * @deprecated in favor of using date-fns functions
+ */
 export const ONE_MINUTE_IN_SECONDS = 60;
+/**
+ * @deprecated in favor of using date-fns functions
+ */
+export const ONE_HOUR_IN_MINUTES = 60;
+/**
+ * @deprecated in favor of using date-fns functions
+ */
 export const ONE_HOUR_IN_SECONDS = ONE_MINUTE_IN_SECONDS * 60;
+/**
+ * @deprecated in favor of using date-fns functions
+ */
 export const ONE_DAY_IN_SECONDS = ONE_HOUR_IN_SECONDS * 24;
 
+/**
+ * @deprecated in favor of using date-fns functions
+ */
 export const FIVE_MINUTES_MS =
   5 * ONE_MINUTE_IN_SECONDS * ONE_SECOND_IN_MILLISECONDS;
+
+/**
+ * @deprecated in favor of using date-fns functions
+ */
 export const ONE_HOUR_IN_MILLISECONDS =
   ONE_SECOND_IN_MILLISECONDS * ONE_HOUR_IN_SECONDS;
-
+/**
+ * @deprecated in favor of using date-fns functions
+ */
 export const SECONDS_TIMESTAMP_MAX_VALUE = 9999999999;
```

Comment:
[NB] Would be good to add the newline of space back between these.

Reply:
added newlines: https://github.com/sparkletown/sparkle/pull/1302/commits/61c8d31c7eecfeae9b1debbdc10b22f31a630afe